manifests/i/ImageMagick/ImageMagick/7.1.0.58/ImageMagick.ImageMagick.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ImageMagick.ImageMagick
+PackageVersion: 7.1.0.58
+PackageLocale: en-US
+Publisher: ImageMagick Studio LLC
+PublisherUrl: https://github.com/ImageMagick/ImageMagick
+PublisherSupportUrl: https://github.com/ImageMagick/ImageMagick/issues
+# PrivacyUrl:
+Author: ImageMagick Studio LLC
+PackageName: ImageMagick
+PackageUrl: https://imagemagick.org
+License: Apache-2.0 Derivative
+LicenseUrl: https://imagemagick.org/script/license.php
+Copyright: Copyright (c) 1999 ImageMagick Studio LLC
+CopyrightUrl: https://imagemagick.org/script/license.php
+ShortDescription: Use ImageMagick to create, edit, compose, or convert digital images.
+# Description:
+# Moniker:
+Tags:
+- bitmap
+- colors
+- convert
+- distort
+- draw
+- effects
+- flip
+- images
+- mirror
+- resize
+- rotate
+- shear
+# Agreements:
+# ReleaseNotes:
+# ReleaseNotesUrl:
```

```suggestion
ReleaseNotesUrl: https://github.com/ImageMagick/Website/blob/main/ChangeLog.md
```
Copy over from previous manifest